Sprint-1/interpret/time-format.js

```diff
@@ -1,24 +1,30 @@
-const movieLength = 8784; // length of movie in seconds
+const movieLength = 999999; // length of movie in seconds
 
 const remainingSeconds = movieLength % 60;
 const totalMinutes = (movieLength - remainingSeconds) / 60;
+console.log("the total" + totalMinutes);
 
 const remainingMinutes = totalMinutes % 60;
 const totalHours = (totalMinutes - remainingMinutes) / 60;
 
 const result = `${totalHours}:${remainingMinutes}:${remainingSeconds}`;
 console.log(result);
 
-// For the piece of code above, read the code and then answer the following questions
+// For the piece of code above, read the ode and then answer the following questions
 
 // a) How many variable declarations are there in this program?
+//  6 variabls
 
 // b) How many function calls are there?
-
+//    1
 // c) Using documentation, explain what the expression movieLength % 60 represents
+//   The remainder (%) operator returns the remainder left over when one operand is divided by a second operand. It always takes the sign of the dividend.fb
```

Comment:
You gave a literal translation of the code, which may not help one understand what the code is trying to calculate. Can you explain in lament terms what the code is trying to calculate?

Reply:
You're right that literal explanations might not help make the purpose of the code clear.
"If I divide one number by another, what’s left over after I take away all the full chunks?"
If you have five cookies and try to divide them into groups of 3, you can make one full group (3 cookies). What's left over is two cookies. So, 5 % 3 = 2.